### ISSUE TITLE
Unable to select multiple elements with "nth" pseudoselectors

### ISSUE THREAD

**Issue by andrewjstalker** [japgolly/scalacss#61]
Currently only accepts an Int value:
```scala
  final case class NthChild(n: Int) extends Pseudo1(s":nth-child($n)", PseudoClass)
  final case class NthLastChild(n: Int) extends Pseudo1(s":nth-last-child($n)", PseudoClass)
  final case class NthLastOfType(n: Int) extends Pseudo1(s":nth-last-of-type($n)", PseudoClass)
  final case class NthOfType(n: Int) extends Pseudo1(s":nth-of-type($n)", PseudoClass)
```

This makes it impossible to select more than one element at a time. Example: http://codepen.io/zakkain/pen/Aqzip

```css
.one li:nth-of-type(1) {
  background: lightsteelblue;
}

.two li:nth-of-type(odd) {
  background: lightsteelblue;
}

.three li:nth-of-type(3n+2) {
  background: lightsteelblue;
```

**Comment by japgolly** [comment]
I wonder if we should simply just make it accept a string. It wouldn't be as safe though...

**Comment by japgolly** [comment]
Fixed via cb0422dfa18acffd2feea4d306561bfeecdbf824
